### PR TITLE
Add current wasm autoflip runner and pbtxt for the demo.

### PR DESCRIPTION
**Pull request by WJ0417**

### CODE REVIEW

File: web_autoflip_demo/autoflip_build/autoflip.embind.cc [hunk 1]
```diff
@@ -0,0 +1,394 @@
+#include <emscripten/bind.h>
```
Comment (parrajustin) [inline web_autoflip_demo/autoflip_build/autoflip.embind.cc:1]: Add license to the top

File: web_autoflip_demo/autoflip_build/autoflip.embind.cc [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+#include <emscripten/html5.h>
+
+#include <string>
+#include <vector>
+
+#include "research/drishti/app/pursuit/wasm/graph_utils.h"
```
Comment (parrajustin) [inline web_autoflip_demo/autoflip_build/autoflip.embind.cc:7]: Remove line

File: web_autoflip_demo/autoflip_build/autoflip.embind.cc (hunk 1 above)
Comment (WJ0417) [reply web_autoflip_demo/autoflip_build/autoflip.embind.cc:15]: Fixed

File: web_autoflip_demo/autoflip_build/autoflip.embind.cc (hunk 2 above)
Comment (WJ0417) [reply web_autoflip_demo/autoflip_build/autoflip.embind.cc:21]: Does this mean to remove line 7?

Comment (WJ0417) [reply web_autoflip_demo/autoflip_build/autoflip.embind.cc]: Fixed

**Pull request merged**